src/composables/addressBook.ts

```diff
@@ -64,19 +64,23 @@ export const useAddressBook = createCustomScopedComposable(() => {
     ])(Object.values(addressBook.value)),
   );
 
+  const addressBookFilteredByProtocol = computed(
+    () => filterAddressBook(Object.values(addressBook.value)),
```

Comment:
I suggest to move the `filterAddressBook` logic here as the function is used only once: here.

Reply:
It's also used in `addressBookFiltered`

Reply:
OK, now I see. So I suggest to rename the initial function to `filterAddressBookByProtocol`.